core/client/app/components/gh-user-invited.js

```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```

Comment:
:+1: :heart:  your mulling technique, I usually just stare out the window... :grin: